lib/CFRPageActions.jsm

```diff
@@ -68,7 +68,12 @@ class PageAction {
     this._showPopupOnClick = this._showPopupOnClick.bind(this);
     this.dispatchUserAction = this.dispatchUserAction.bind(this);
 
-    this._l10n = new DOMLocalization(["browser/newtab/asrouter.ftl"]);
+    this._l10n = new DOMLocalization([
+      "browser/newtab/asrouter.ftl",
+      "browser/branding/brandings.ftl",
+      "browser/branding/sync-brand.ftl",
+      "branding/official/brand.ftl",
```

Comment:
You should be able to test this by triggering a cfr message from newtab devtools. Pretty sure this will fail because the packaged brand.ftl has no `official/`

Reply:
oh hmm - that was the only place I saw `-brand-product-name` defined; I can see what happens without it!

Reply:
You'll need to include brand.ftl. Just don't have the `official/` in the path. The source directory is to track different versions of the strings but the packaged-into-firefox version only picks one. See other uses:

https://searchfox.org/mozilla-central/search?q=%2Fbrand.ftl&path=.html

Reply:
Unfortunately triggering CFRs from DevTools is a little hit and miss...

Reply:
hit and miss how?